appium/webdriver/webdriver.py

```diff
@@ -400,6 +402,23 @@ def current_activity(self):
         """
         return self.execute(Command.GET_CURRENT_ACTIVITY)['value']
 
+    def wait_activity(self, ac, retry=10, interval=1):
```

Comment:
I would prefer `app_activity` over `ac`, `retries` over `retry`, and something like `timeout` over `interval`, to keep things in line with the naming conventions throughout the module.